openmeter/ent/schema/price.go

```diff
@@ -0,0 +1,54 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/dialect"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+
+	"github.com/openmeterio/openmeter/openmeter/productcatalog/plan"
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+type Price struct {
```

Comment:
Let's prefix these like `SubscriptionPrice`

Reply:
deleted it